### PR TITLE
l10n: sync ar de_DE fr_FR nb_NO nl translations

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

These are incomplete translations to release with 0.4.4 and should
be prominently marked as "beta". The localizationlab reviewers are:

* ar: not reviewed
* de_DE: https://weblate.securedrop.club/user/Atalanttore
* fr_FR: https://weblate.securedrop.club/user/french.coordinator
* nb_NO: https://weblate.securedrop.club/user/kingu
* nl: https://weblate.securedrop.club/user/AnneM, https://weblate.securedrop.club/user/kwadronaut

See docs/development/i18n.rst for more information on how these files
are updated.

### Translations are technically correct

* vagrant up development
* vagrant ssh development
* cd /vagrant/securedrop
* ./manage.py translate --compile
* this checks there are no broken placeholders in the translated strings
<pre>
PAGE_LAYOUT_LOCALES='ar,de_DE,fr_FR,nb_NO,nl' pytest -v --page-layout tests/pages-layout
</pre>

### Translations are compiled and properly packaged

* make build-debs # includes a check that the *.mo files are present
* vagrant provision app_staging
* go to the source interface THS 
* verify there is no language menu
* add to config.py
<pre>
SUPPORTED_LOCALES = ['en_US', 'ar', 'de_DE', 'fr_FR', 'nb_NO', 'nl']
</pre>
* go to the source interface THS
* click on each language in the menu and verify the strings change accordingly # verifies apparmor rules work

## Deployment

New files are packaged (the .mo) and created during the build process. They are not used by default and should not interfere with existing installations.

### If you made changes to the app code:

- [x] Unit and functional tests pass on the development VM

### CODE REVIEW

**Comment by dachary** [comment]
https://circleci.com/gh/freedomofpress/securedrop/2795?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

<pre>
"Timeout when waiting for ec2-52-53-224-199.us-west-1.compute.amazonaws.com:22"
</pre>

**Comment by dachary** [comment]
https://circleci.com/gh/freedomofpress/securedrop/2807?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Tons of errors, ends with a reboot timeout. But maybe it's because some bad happened earlier. Only I can't see why there is too much noise. Retrying.

**Comment by dachary** [comment]
@redshiftzero it is ready for review. Big moment :-)

File: securedrop/i18n.py [hunk 1]
```diff
@@ -101,13 +101,13 @@ def get_text_direction(locale):
...
-    """Return SUPPORTED_LOCALES from config.py. It is missing return all
-    locales found in the translations directory.
+    """Return SUPPORTED_LOCALES from config.py. If it is missing return
+    the default locale.
 
     """
 
     if not supported:
-        return sorted(locales)
+        return [default_locale or 'en_US']
```
Comment (redshiftzero) [inline securedrop/i18n.py]: :+1:

File: securedrop/translations/ar/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -0,0 +1,1256 @@
...
+#: source_templates/generate.html:29
+msgid ""
+"Because we use none of the traditional means to track users of our "
+"<strong>SecureDrop</strong>\n"
+" service, using this codename with future visits is the  only way we have to "
+"communicate with you, should we have\n"
+" questions or are interested in additional documents. Unlike passwords, "
+"there is no way to retrieve a lost codename."
+msgstr ""
+"<strong> 1SecureDrop</strong> 2 لأننا لا نستخدم أي من الوسائل التقليدية "
```
Comment (redshiftzero) [inline securedrop/translations/ar/LC_MESSAGES/messages.po]: There are a couple of nits like this `1SecureDrop` in the Arabic translation. I've verified all page layout tests pass, so let's merge in what we have so far and see if we can wrangle up someone to correct any issues in the next few days. Else we will ship what we have.

**Comment by dachary** [comment]
@redshiftzero <3 \o/ /o/ \o\ /o\ :-)

**Pull request merged**